README.md

```diff
@@ -13,23 +13,23 @@ Detailed activity reports can be generated across multiple masters using the fol
 * [merge-user-activity-monitoring-reports.sh](./merge-user-activity-monitoring-reports.sh)
     * generates a unique report called `/out/aggregated-user-activity.json`
     * `/out/aggregated-user-activity.json` contains a list of all `authAccess` entries and a list of all `scmAccess` entries
-    * The entries contain the `firstDayOfWeek`, the `name` and the `server` entry like:
+    * The entries contain the `firstDayOfMonth`, the `name` and the `server` entry like:
     ```
     {
         "server": {
             "url": "https://cd.wordsmith.beescloud.com/teams-front-team/",
             "id": "21e47a96ced17c97e72f91cd7f7c6a0f"
         },
-        "firstDayOfWeek": "2018-10-28",
+        "firstDayOfMonth": "2018-10-01",
         "name": "simon"
     }
     ```
 * [count-user-activity.sh](./count-user-activity.sh)
     * displays on the stdout the summary of the users counted from `/out/aggregated-user-activity.json`:
     ```
-    1 user(s) counted as Authenticated from 2018-10-28 to 2019-02-03
-    3 user(s) counted as SCM contributor from 2018-10-28 to 2018-12-30
-    4 user(s) counted as CloudBees Core users from 2018-10-28 to 2019-02-03
+    1 user(s) counted as Authenticated from 2018-10-01 to 2019-02-01
```

Comment:
Shouldn't the end date be `2018-31-12` if you count by months? Or maybe I misunderstood this all month thing.